File: IT/1/Variabili, operatori, strutture di controllo/Python/whileTest.py
# file whileTest.py

from __future__ import print_function
from sys import stdout
from os import linesep

# inizializza la variabile contatore

c = 0

# itera per cinque volte

while c < 5:
	print(c, end=" ")
	c = c + 1

# fine del ciclo

# va a capo
stdout.write(linesep)
print("Fine del ciclo.")

File: IT/1/Variabili, operatori, strutture di controllo/Java/WhileTest.java
/* file WhileTest.java */

public class WhileTest {
	public static void main(String [] args) {
		/* inizializza la variabile contatore */

		int c = 0;

		/* itera per cinque volte */

		while (c < 5) {
			System.out.print(c + " ");
			c++;
		}

		/* fine del ciclo */

		System.out.print(System.lineSeparator());
		System.out.println("Fine del ciclo.");
	}
}

